### ISSUE TITLE
Travis can't find package memisc

### ISSUE THREAD

**Issue by DCMSstats** [DCMSstats/eesectors#83]
With implementation of #82 Travis fails due to the use of package memisc.

```
Setting up build cache
$ export CASHER_DIR=$HOME/.casher
0.05s$ Installing caching utilities
0.01s
2.01sattempting to download cache archive
fetching PR.82/cache-linux-precise-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855--R-3.3.2.tgz
found cache
0.00s
4.83sadding /home/travis/build/ukgovdatascience/eesectors/packrat/src to cache
adding /home/travis/build/ukgovdatascience/eesectors/packrat/lib to cache
creating directory /home/travis/build/ukgovdatascience/eesectors/packrat/lib
package
Setting up package cache
0.00s
3.52sadding /home/travis/R/Library to cache
creating directory /home/travis/R/Library
R-session-info
R session information
$ Rscript -e 'sessionInfo()'
R version 3.3.2 (2016-10-31)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu precise (12.04.5 LTS)
locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       
attached base packages:
[1] stats     graphics  grDevices utils     datasets  base     
loaded via a namespace (and not attached):
[1] tools_3.3.2     packrat_0.4.8-1
install
0.35s$ R -e "0" --args --bootstrap-packrat
R version 3.3.2 (2016-10-31) -- "Sincere Pumpkin Patch"
Copyright (C) 2016 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)
R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.
  Natural language support but running in an English locale
R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.
Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.
> 0
[1] 0
> 
> 
R-build
Building package
Building with: R CMD build 
3.38s$ R CMD build  .
* checking for file ‘./DESCRIPTION’ ... OK
* preparing ‘eesectors’:
* checking DESCRIPTION meta-information ... OK
* checking for LF line-endings in source and make files
* checking for empty or unneeded directories
Removed empty directory ‘eesectors/packrat’
* looking to see if a ‘data/datalist’ file should be added
* building ‘eesectors_0.1.0.tar.gz’
R-check
Checking package
Checking with: R CMD check eesectors_0.1.0.tar.gz --as-cran --no-manual --no-manual --no-manual --no-manual
2.01s$ R CMD check "${PKG_TARBALL}" --as-cran --no-manual --no-manual --no-manual --no-manual; CHECK_RET=$?
* using log directory ‘/home/travis/build/ukgovdatascience/eesectors/eesectors.Rcheck’
* using R version 3.3.2 (2016-10-31)
* using platform: x86_64-pc-linux-gnu (64-bit)
* using session charset: UTF-8
* using options ‘--no-manual --as-cran’
* checking for file ‘eesectors/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘eesectors’ version ‘0.1.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... ERROR
Package required but not available: ‘memisc’
See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
* DONE
Status: 1 ERROR
See
  ‘/home/travis/build/ukgovdatascience/eesectors/eesectors.Rcheck/00check.log’
for details.
0.58schecking package dependencies ... ERROR
Package required but not available: ‘memisc’
See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

Any ideas @ivyleavedtoadflax?

**Comment by ivyleavedtoadflax** [comment]
Might need to add this to your packrat library

**Comment by ivyleavedtoadflax** [comment]
Resolved in #82.
